Use 8.8.8.8 as a static test address

coolparadox opened the pull request:
Also contains:

- Introductory documentation about the library (for godoc)
- Test example of LookupAsn usage (also shown in documentation)
- Godoc badge in github project's frontpage

Fixes: #4

coolparadox committed:
Using 8.8.8.8 as the test IP address.

References #4

coolparadox committed:
Added example test of LookupAsn

References #4

coolparadox committed:
Badge reference to web documentation added to github frontpage.

References #4

coolparadox committed:
Introductory documentation about the library.

References: #4

sajal commented:
I think that the tests should do some assertion on the returned data. Things like.
1. Length of the returned strings are > 0
2. The ASN is either AS3356 or AS15169 and ASN Name contains `Google`  or `Level 3` .

coolparadox commented:
+1

coolparadox committed:
Test assertions on expected return values for all lookups.

Now all lookup tests fail if returned ASN is not AS15169 (with "Google Inc" as a substring of its description) or AS3356 (with "Level 3 Communications") as a substring of its description. These are expected for 8.8.8.8 address.

References #4
